<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportVisitsByClientTypeSetupDialog</class>
 <widget class="QDialog" name="ReportVisitsByClientTypeSetupDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>параметры отчёта</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="lblCitizen">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Житель</string>
     </property>
     <property name="buddy">
      <cstring>cmbOrgStructure</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lblBegDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Дата &amp;начала периода</string>
     </property>
     <property name="buddy">
      <cstring>edtBegDate</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QLabel" name="lblMKBTo">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>по</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="lblEventType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Тип обращения</string>
     </property>
     <property name="buddy">
      <cstring>cmbEventType</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="lblEndDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Дата &amp;окончания периода</string>
     </property>
     <property name="buddy">
      <cstring>edtEndDate</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="CICDCodeEditEx" name="cmbMKBFrom">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="lblMKBRange">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Диапазон диагнозов:</string>
     </property>
     <property name="buddy">
      <cstring>cmbOrgStructure</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="lblMKBFrom">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>с</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="5">
    <widget class="QDateEdit" name="edtBegDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="5">
    <widget class="QComboBox" name="cmbIsPrimary">
     <item>
      <property name="text">
       <string>Не заполнять</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Первичное</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Повторное</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="chbMKBFilter">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="4" colspan="2">
    <spacer name="vSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="lblOrgStructure">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Подразделение</string>
     </property>
     <property name="buddy">
      <cstring>cmbOrgStructure</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="5">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="lblPrimaryStatus">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Вид обращения</string>
     </property>
     <property name="buddy">
      <cstring>cmbEventType</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="5">
    <widget class="QDateEdit" name="edtEndDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="6">
    <widget class="CICDCodeEditEx" name="cmbMKBTo">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="5">
    <widget class="CRBComboBox" name="cmbEventType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="5">
    <widget class="QComboBox" name="cmbCitizen">
     <item>
      <property name="text">
       <string>Не заполнять</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Житель Санкт-Петербурга</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Житель Лен. области</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Иногородний житель</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Иностранец</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Сельский житель</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CICDCodeEditEx</class>
   <extends>QFrame</extends>
   <header>library/ICDCodeEdit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtBegDate</tabstop>
  <tabstop>edtEndDate</tabstop>
  <tabstop>cmbEventType</tabstop>
  <tabstop>cmbOrgStructure</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportVisitsByClientTypeSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportVisitsByClientTypeSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chbMKBFilter</sender>
   <signal>toggled(bool)</signal>
   <receiver>lblMKBRange</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chbMKBFilter</sender>
   <signal>toggled(bool)</signal>
   <receiver>lblMKBFrom</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>21</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chbMKBFilter</sender>
   <signal>toggled(bool)</signal>
   <receiver>lblMKBTo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chbMKBFilter</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbMKBFrom</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chbMKBFilter</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbMKBTo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
